src/main/scala/nl.knaw.dans.easy.solr4files/components/FileItem.scala

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.net.URL
+
+import nl.knaw.dans.easy.solr4files.SolrLiterals
+
+import scala.xml.Node
+
+case class FileItem(bag: Bag, ddm: DDM, xml: Node) {
+
+  // see ddm.xsd EasyAccessCategoryType
+  private def datasetAccessitbleTo = ddm.accessRights match {
+    // @formatter:off
+    case "OPEN_ACCESS"                      => "ANONYMOUS"
+    case "OPEN_ACCESS_FOR_REGISTERED_USERS" => "KNOWN"
+    case "GROUP_ACCESS"                     => "RESTRICTED_GROUP"
+    case "REQUEST_PERMISSION"               => "RESTRICTED_REQUEST"
+    case "NO_ACCESS"                        => "NONE"
+    case _                                  => "NONE"
+    // @formatter:off
+  }
+  val accessRights: String = ( xml \ "accessRights").map(_.text.trim).mkString
```

Comment:
You're sure about this `mkString`? `["abc", "def", "ghi"].mkString == "abcdefghi"`